src/Bridge/Doctrine/Orm/Extension/OrderExtension.php

```diff
@@ -37,6 +37,7 @@ public function applyToCollection(QueryBuilder $queryBuilder, QueryNameGenerator
         $classMetaData = $queryBuilder->getEntityManager()->getClassMetadata($resourceClass);
         $identifiers = $classMetaData->getIdentifier();
 
+        //@TODO Why?
         if (null !== $this->order) {
             foreach ($identifiers as $identifier) {
                 $queryBuilder->addOrderBy('o.'.$identifier, $this->order);
```

Comment:
We want results ordering to be deterministic. If no `ORDER BY` is specified in the SQL, the database is free to return rows in any order.